cmd/kubeadm/app/util/version.go

```diff
@@ -251,7 +251,7 @@ func validateStableVersion(remoteVersion, clientVersion string) (string, error)
 	if verClient.Major() < verRemote.Major() ||
 		(verClient.Major() == verRemote.Major()) && verClient.Minor() < verRemote.Minor() {
 		estimatedRelease := fmt.Sprintf("stable-%d.%d", verClient.Major(), verClient.Minor())
-		klog.Infof("remote version is much newer: %s; falling back to: %s", remoteVersion, estimatedRelease)
+		klog.InfoS("remote version is much newer, falling back", "remoteVersion", remoteVersion, "backtoVersion", estimatedRelease)
 		return estimatedRelease, nil
```

Comment:
As if remoteVersion and estimatedRelease are regular string. So we needn't change this line.

Reply:
We should close this PR, i think.